port/win32/omrintrospect.c

```diff
@@ -176,13 +175,8 @@ suspend_all_preemptive(struct PlatformWalkData *data)
 		passA = !passA;
 	} while (a_count != b_count);
 
-	if (a_count != b_count) {
-		goto error_return;
```

Comment:
I agree I do not see how the code would exist the loops above without the 2 counts being equal to one another without just returning.   I am going to open an issue for this function though.  If the thread count did change I do not see how the loop would ever exit since the counts are never reset.